suite-native/module-accounts-import/src/components/AssetsOverview.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import { CryptoIcon, IconName } from '@trezor/icons';
+import {
+    Box,
+    Card,
+    Input,
+    InputWrapper,
+    SelectableListItem,
+    Text,
+    VStack,
+} from '@suite-native/atoms';
+import { AccountInfo } from '@trezor/connect';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+
+export type DummyDevice = { icon: IconName; title: string; value: string };
+
+type AssetsOverviewProps = {
+    accountInfo: AccountInfo;
+    selectedDevice: DummyDevice | undefined;
+    assetName: string;
+    onSelectDevice: (device: DummyDevice) => void;
+    onAssetNameChange: (value: string) => void;
+};
+
+export const dummyDevices: Array<DummyDevice> = [
```

Comment:
I guess you will remove these in follow-up in favor of some custom `Imported` device?